### PR TITLE
add Cleaning module

### PR DESCRIPTION
**Pull request by ildyria**
Built on top of #2348

### CODE REVIEW

File: app/Livewire/Components/Modules/Maintenance/Cleaning.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+		$count = 0;
+		foreach (new \DirectoryIterator($this->path) as $fileInfo) {
+			if ($fileInfo->isDot()) {
+				continue;
+			}
+			if (in_array($fileInfo->getFilename(), $this->skip, true)) {
+				continue;
+			}
+			$count = 1;
```
Comment (d7415) [inline app/Livewire/Components/Modules/Maintenance/Cleaning.php]: Could we just return `false` from here?

**Review by d7415 — APPROVED** [review]
One query, otherwise looks ok.

File: app/Livewire/Components/Modules/Maintenance/Cleaning.php (hunk 1 above)
Comment (ildyria) [reply app/Livewire/Components/Modules/Maintenance/Cleaning.php]: true, I should be using a boolean here.

Comment (ildyria) [reply app/Livewire/Components/Modules/Maintenance/Cleaning.php]: see [4dcebc5](https://github.com/LycheeOrg/Lychee/pull/2349/commits/4dcebc5c89d8d4f3d86706561d26509c167d22f7)

**Pull request merged**